AutonomousDemo/Position.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AutonomousDemo
+{
+    public class Position : IPosition
+    {
+        public double XCoordinate { get; set; } = 0;
+        public double YCoordinate { get; set; } = 0;
+
+        public Position(double xCoordinate, double yCoordinate)
+        {
+            this.XCoordinate = xCoordinate;
+            this.YCoordinate = yCoordinate;
+        }
+        public Position TravelToPosition(double meters, Velocity velocity)
```

Comment:
If a velocity is a speed and a direction, why is there a function to travel an arbitrary distance in that direction?